backend/packages/Upgrade/src/api/services/SegmentService.ts

```diff
@@ -85,38 +85,35 @@ export class SegmentService {
       const allExperimentSegmentsInclusion = await this.getExperimentSegmenInclusionData();
       const allExperimentSegmentsExclusion = await this.getExperimentSegmenExclusionData();
```

Comment:
These three sequential awaits should be executed parallelly.

Reply:
resolved